resources/js/store/index.js

```diff
@@ -13,12 +13,20 @@ const getters = {
 const actions = {
     add: function ({commit}, todo) {
         commit('ADD', todo);
+    },
+
+    delete: function ({commit}, todo) {
+        commit('DELETE', todo);
     }
 };
 
 const mutations = {
     ADD: function (state, todo) {
         state.todos.push(todo)
+    },
+
+    DELETE: function (state, todo) {
+        state.todos.filter(todo)
```

Comment:
Does ths work?
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/filter

JavaScript's array filter seems to take in a function in order to define the predicate on which to filter the collection.

Also, I don't believe JavaScript's array filter modifies in place so you'll have to reassign the value to `state.todos`